seed/challenges/00-getting-started/getting-started.json

```diff
@@ -21,15 +21,15 @@
           ""
         ],
         [
-          "//i.imgur.com/D7Y5luw.jpg",
-          "A graph of the rate of job growth against growth in computer science degree graduates. There are 1.4 million jobs and only 400 thousand people to fill them.",
-          "There are thousands of coding jobs currently going unfilled, and the demand for coders grows every year.",
+          "//i.imgur.com/WD3STY6.jpg",
+          "Photos of three campers who've gotten jobs after learning to code at Free Code Camp.",
```

Comment:
I was thinking let's not even get into that debate. We could simply write "who've received job offer"